src/client/modules/content/widgets/data_table/ProjectCollaboratorTable.jsx

```diff
@@ -0,0 +1,200 @@
+                    organizationCollaborators.push({
+                        name: organization,
+                        read: organizationsWithAccess[organization].read,
+                        write: organizationsWithAccess[organization].write,
+                        delete: organizationsWithAccess[organization].delete,
+                        inOrg: organizationsWithAccess[organization].inOrg
+                    });
+                });
+
+                this.setState({
+                    userCollaborators: userCollaborators.sort(sortObjectArrayByField('name')),
+                    organizationCollaborators: organizationCollaborators.sort(sortObjectArrayByField('name'))
+                });
+            }
+
+        });
+    }
+
+    handleOrderEntries() {
+        if (this.state.display === 1) {
+            this.setState({
+                userCollaborators: this.state.numTimesClicked % 2 === 0 ?
+                    this.state.userCollaborators.sort(sortObjectArrayByField('name')).reverse() :
+                    this.state.userCollaborators.sort(sortObjectArrayByField('name')),
+                numTimesClicked: this.state.numTimesClicked + 1
+            });
+        } else {
+            this.setState({
+                organizationCollaborators: this.state.numTimesClicked % 2 === 0 ?
+                    this.state.organizationCollaborators.sort(sortObjectArrayByField('name')).reverse() :
+                    this.state.organizationCollaborators.sort(sortObjectArrayByField('name')),
+                numTimesClicked: this.state.numTimesClicked + 1
+            });
+        }
+    }
+
+    handleRevoke(event) {
+        this.props.restClient.projects.removeRightsToProject(this.props.ownerId,
+            this.props.projectName,
+            event.target.id)
+            .then(() => {
+                this.retrieveCollaborators(); // Re-render after revoking rights
+            });
+    }
+
+    handleTableSwitch(event) {
+        let newDisplayNum = event.target.innerHTML === 'Users' ? 1 : 2;
+
+        if (newDisplayNum !== this.state.display) {
+            this.setState({
+                display: newDisplayNum
+            });
+        }
+    }
+
+    render() {
+
+        let dataTableData = {
+            categories: {
+                users: [
+                    {id: 1, name: 'UserID:'},
+                    {id: 2, name: 'Rights (RWD)'}
+                ],
+                organizations: [
+                    {id: 1, name: 'OrganizationID:'},
+                    {id: 2, name: 'Rights(RWD)'}
+                ]
+            },
+            dualTable: {
+                show: true,
+                options: ['Users', 'Organizations']
+            }
+        };
+
+        return (
+            <div>
+
+                <DataTable categories={dataTableData.categories.users}
+                           display={this.state.display}
+                           dualTable={dataTableData.dualTable}
```

Comment:
I think it's only needed to keep the buttons in the data table's header. I guess it could be taken out if the data table were split into even more components - Header, Body, (Already has Pagination)

All it does is add and bind the buttons to the event handler that switches the display which renders the other table right now..